cookbooks/bcpc-hadoop/recipes/hadoop_patch.rb

```diff
@@ -0,0 +1,55 @@
+directory '/usr/local/hadoop' do
+  owner 'root'
+  group 'root'
+  mode '0755'
+  action :create
+end
+remote_file '/usr/local/hadoop/hadoop-common-2.7.1.jar' do 
+  #source 'http://10.0.100.3/hadoop-common-2.7.1.jar'
+  source "#{get_binary_server_url}/hadoop-common-2.7.1.jar"
+  checksum = 'sha256:9ed127f5a9a21c2f3efced7fd73e36d782b73b8eb9820c7ebef5169a94ded8f0'
+  owner 'root'
+  group 'root'
+  mode '0644'
+  action :create
+end
+# backup the previous file
```

Comment:
This is a temporary fix until the new release comes out.  If we get a new release, then this patch needs to be removed.  We could put in a guard if you think that is necessary.